Makefile

```diff
@@ -63,11 +62,13 @@ $(PY36):
 $(PYPY):
 	scripts/retry.sh scripts/install.sh pypy
 
-$(TOOL_VIRTUALENV): $(PY34)
+$(TOOL_VIRTUALENV): $(PY34) requirements/tools.txt
+	rm -rf $(TOOL_VIRTUALENV)
```

Comment:
The `documentation` make task installs sphinx and sphinx-rtd-theme in this virtualenv, then calls this Make task. I think you’re blowing away Sphinx after it’s installed. Needs fixing before we can get a clean run of the tests.